mirascope/core/openai/call_params.py

```diff
@@ -11,17 +13,36 @@
 
 from ..base import BaseCallParams
 
+if TYPE_CHECKING:
+    from openai.types.chat.chat_completion_audio_param import (  # pyright: ignore [reportMissingImports]
+        ChatCompletionAudioParam,
+        ChatCompletionModality,  # pyright: ignore [reportAttributeAccessIssue]
+    )
+else:
+    try:
+        from openai.types.chat.chat_completion_audio_param import (  # pyright: ignore [reportMissingImports]
+            ChatCompletionAudioParam,
+            ChatCompletionModality,
+        )
+    except ImportError:
+
+        class ChatCompletionAudioParam: ...
+
+        class ChatCompletionModality: ...
```

Comment:
The audio model is supported only the latest OpenAI SDK.
Therefore, we import it here to support older sdk versions.